core/domain/state_domain.py

```diff
@@ -1406,7 +1416,12 @@ def validate(self, exp_param_specs_dict, allow_null_interaction):
         self.recorded_voiceovers.validate(content_id_list)
 
     def get_training_data(self):
-        """Retrieves training data from the State domain object."""
+        """Retrieves training data from the State domain object.
+
+        Returns:
+            list(dict). Each dict contains the index and the answers from
```

Comment:
The docstring suggests that the dict is formatted as { index : answers_from_the_answer_group } which I think isn't what you mean to convey. Enumerate the list of keys explicitly, and give the types and descriptions of their values. See e.g. get_data_for_unfinished_jobs() in core/jobs.py.

Reply:
Done